setup.py

```diff
@@ -8,13 +8,18 @@
     name="wps_tools",
     description="A collection of modules used to help create wps processes",
     keywords="wps pywps wps_tools",
-    version='.'.join(str(d) for d in __version__),
+    version=".".join(str(d) for d in __version__),
     url="https://github.com/pacificclimate/wps-tools",
     author="Eric Yvorchuk",
     author_email="eyvorchuk@uvic.ca",
     packages=["wps_tools"],
+    package_data={
+        "wps-tools": ["tests/data/*.nc", "processes/*.py"],
```

Comment:
This may be a dumb question since I am not very familiar with `setup.py`, but do we need to include processes in `pacakge_data`.

Reply:
I don't actually know. I put this in here so that in `test_tools.py`, I could ensure that I'm able to import `SayHello` while in a subdirectory of `wps-tools`.